german/Leichen/Leichen.md

```diff
@@ -1,7 +1,7 @@
 Sie schauen ängstlich um sich, sehen aber keine andere Menschenseele außer den beiden
 völlig durchlöcherten Leichnamen neben sich, deren Blut sie mittlerweile grosszügig an die
 Lebewesen im See gespendet haben. Ihr Boot ist bereits in leichter Schieflage und hastig
-werfen die beiden Leichen aus dem Boot.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.